wordpress-mobile/WordPress-iOS
# Update SIWA button appearance

Fixes #13620 
Auth PR: https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/213

This uses the Auth changes to display the updated Login & Signup buttons, as described in the Auth PR.

To test:

---
- If logged in, log out.
- Select `Log in` and `Sign up`.
- Verify the three buttons all have the same appearance. (Button size, font, background color)

| Log In | Sign Up |
|--------|-------|
| ![login](https://user-images.githubusercontent.com/1816888/77195446-418c0700-6aa7-11ea-8c8c-5b7368e41db0.png) | ![signup](https://user-images.githubusercontent.com/1816888/77195463-481a7e80-6aa7-11ea-96bb-db4991693efa.png) |

---
- Toggle light/dark modes.
- Verify the colors update accordingly. Specifically:
  - In dark mode, the buttons and button view backgrounds are black.
  - The Apple logo updates.

| Log In | Sign Up |
|--------|-------|
| ![login_dark](https://user-images.githubusercontent.com/1816888/77195491-4fda2300-6aa7-11ea-8cd2-acc3127f3bc0.png) | ![signup_dark](https://user-images.githubusercontent.com/1816888/77195507-56689a80-6aa7-11ea-8938-88cdba321af3.png) |

PR submission checklist:
